run.sh

```diff
@@ -149,8 +153,8 @@ do
 
   # Don't run presence detection if TTL is 0
   if [ $PRESENCE_DETECTION_TTL -gt 0 ] ; then
-    counter=$(expr $counter + 1)
-    if [[ $counter -gt 90 ]]; then
+    counter=$((counter + 1))
```

Comment:
[[ ]] is actually supported in ash.

I added in .shellscheck global rule to ignoe - we have 1 place it's obviously useful, when checking the failed output from tesla-control.